manifests/init.pp

```diff
@@ -18,7 +18,19 @@
 # $plugin_version::             foreman plugins version, it's passed to ensure parameter of plugins package resource
 #                               can be set to 'latest', 'present',  'installed', 'absent'.
 #
-# $port::                       Port on which will foreman proxy listen
+# $port::                       Port to listen on (deprecated in favor of $ssl_port and $http_port)
+#                               type:integer
+#
+# $http::                       Enable HTTP
+#                               type:boolean
+#
+# $http_port::                  HTTP port to listen on (if http is enabled)
+#                               type:integer
+#
+# $ssl::                        Enable SSL, ensure feature is added with "https://" protocol if true
+#                               type:boolean
+#
+# $ssl_port:                    HTTPS port to listen on (if ssl is enabled)
```

Comment:
Sorry, ignore my previous comment.  This is missing a second colon after the parameter name.